src/components/menu-right/ContGateTurnEditor.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+  <div class="my-1 px-1">
+    <b-field>
+      <b-select
+        v-model="selectedMarchType"
+        size="is-small"
+        data-test="cont-gate-turn--march-type"
+      >
+        <option
+          v-for="(marchType, marchTypeIndex) in marchTypeOptions"
+          :key="`cont-gate-turn-${continuityIndex}-${marchTypeIndex}`"
+          :value="marchType.value"
+          :selected="selectedMarchType == marchType.value"
+        >
+          {{ marchType.friendlyName }}
+        </option>
+      </b-select>
+      <b-numberinput
+        :controls="false"
+        v-model="selectedDuration"
+        size="is-small"
+        min="0"
+        data-test="cont-gate-turn--duration"
+      />
+      <b-numberinput
+        :controls="false"
+        v-model="selectedAngle"
+        size="is-small"
+        data-test="cont-gate-turn--angle"
+      />
+      <p class="control" v-if="canDelete">
+        <b-button
+          icon-left="delete"
+          size="is-small"
+          @click="deleteContinuity"
+          data-test="cont-etf-static--delete"
+        />
+      </p>
+      <!-- TODO: Proper centering tool -->
```

Comment:
You've probably seen me say this elsewhere; I tend to think that if there is a TODO in the code, it should be accompanied by an issue in the issue tracker. If this is already being tracked, it may be helpful to pin the issue number into this comment so that it's easy to connect the two.

Reply:
Agreed, but if you meant to do this in a later commit then it's okay! I tend do leave TODOs for things that I still need to get to in my PRs.

Reply:
I'd intended this for a future PR, so I'll make an issue right now!

Reply:
Can you put the issue number in the code comment?